pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+testpaths = "convert2rhel/unit_tests"
+addopts = --cov-fail-under=83 --tb=native
+;log_cli = false
+;log_cli_level = DEBUG
+;log_cli_format = "| %(asctime)s | %(name)s | %(levelname)s | %(filename)s | %(message)s"
```

Comment:
Is this commented out intentionally?

Reply:
yes, I'd like to give a quick path to enable live logging when running tests